src/views/IndividualTeamView.vue

```diff
@@ -97,11 +159,66 @@ export default {
         },
       });
     },
+    joinThisTeam() {
```

Comment:
Here's how I'd do this: 
```
joinTeam(id).then((response1) => {
    return getTeam(response1.teamid)
}).then((response2) => {
    this.team = response2.data;
    this.loaded = true;
}).catch((e) => {
    ...
});
```

When you return a 2nd promise in the first then, you can just chain another then to access the resolution of that second promise (wow that was a wordy sentence)

basically, you can chain this as long as you want, and that's easier to work with than nesting it because you have one `catch` that catches everything (it'll catch errors in both the first and second promises)